packages/client/lib/rpc/modules/engine.ts

```diff
@@ -363,11 +330,7 @@ export class Engine {
       } catch (error) {
         const validationError = `Invalid tx at index ${index}: ${error}`
         this.config.logger.error(validationError)
-        const latestValidHash = await validHash(
-          toBuffer(payloadData.parentHash),
-          this.validBlocks,
-          this.chain
-        )
+        const latestValidHash = await validHash(toBuffer(payloadData.parentHash), this.chain)
```

Comment:
Is `payloadData.parentHash` ever anything but a hex string?  If not, I wonder if there's an opportunity to add an explicit `hexToBuffer` utility method in `ethereumjs-util` that we could leverage here?  I'm remembering past conversations we've had about when to use `toBuffer` versus a more explicit utility method.

Reply:
I think using toBuffer is fine for hex string to buffer conversion, don't really see the benefit of using something specifically for 0x-prefixed hex string to buffer conversion when toBuffer is meant to cover that case.